datafusion/core/tests/sql/explain_analyze.rs

```diff
@@ -694,7 +694,7 @@ async fn test_physical_plan_display_indent() {
         "            RepartitionExec: partitioning=Hash([Column { name: \"c1\", index: 0 }], 9000)",
         "              AggregateExec: mode=Partial, gby=[c1@0 as c1], aggr=[MAX(aggregate_test_100.c12), MIN(aggregate_test_100.c12)]",
         "                CoalesceBatchesExec: target_batch_size=4096",
-        "                  FilterExec: c12@1 < CAST(10 AS Float64)",
+        "                  FilterExec: c12@1 < 10",
```

Comment:
This is the physical plan, which no longer contains a cast here because the logical plan optimized out the cast of a literal value.

Reply:
🎉  -- which I think is a good example of the value of this pass